src/main/java/org/folio/marccat/resources/domain/FixedField.java

```diff
@@ -448,10 +465,48 @@
   @JsonProperty("formatCode")
   private String formatCode;
 
+  @JsonProperty("subjectDescriptor")
```

Comment:
Aren't these annotations superfluous if using the same value as the member name?

Reply:
I have removed them.